index.js

```diff
@@ -24,24 +29,43 @@ addTaskButton.addEventListener("click", () => {
     createdAt: new Date().toISOString(),
   });
 
-  renderTasks();
+  renderTasks(tasks);
+});
+
+searchButton.addEventListener("click", () => {
+  filterTasksByTitle(sreachInput.value);
 });
 
-function renderTasks() {
+// sortTaskDropdown.addEventListener("change", () => {
+//   const sortType =
+//     sortTaskDropdown.options[sortTaskDropdown.selectedIndex].value;
+
+//   sortTasks(sortType);
+// });
+
+timeFilterDropdown.addEventListener("change", () => {
+  const selectedTime =
+    timeFilterDropdown.options[timeFilterDropdown.selectedIndex].value;
```

Comment:
Why not just? https://jsfiddle.net/blacksrc/fhj8L6sy/2/

Reply:
I didnt know that.
Its greattttt thanks ^^